src/Sylius/Bundle/AdminBundle/Form/Model/PasswordResetRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Form\Model;
+
+class PasswordResetRequest
+{
+    private ?string $email;
+
+    public function __construct()
+    {
+        $this->email = null;
+    }
+
+    public function getEmail(): ?string
+    {
+        return $this->email;
+    }
+
+    public function setEmail(?string $email): void
```

Comment:
do we need this setter? this class looks like VO, can we set this email in constructor?

Reply:
@AdamKasp Before executing `$form->handleRequest($request);` the object is "empty" so forcing to set e-mail as a constructor's parameter in my opinion is not the best solution